Gameplay

rchorngren opened the pull request:
Adds the functionality to play the game
 - after entering their name the player can start the game by clicking the Play-button
 - during the game the player selects if they think the next card will be higher or lower than the card currently shown
 - if correct, they  increase their score and guess again (by pressing the another-button, which is a placeholder for now)
 - if incorrect, they are navigated to the highscore screen (which is a future task)

rchorngren committed:
Added game activity and player class

Commit:
started on card game functionality

rchorngren committed:
Adjusted positions of views in GameActivity
Made sure random numbers cannot be the same in order to avoid a draw

rchorngren committed:
Added (placeholder) ability to start another game

Commit:
added score counter

rchorngren committed:
Remove the possibility of a draw when playing another game

rchorngren committed:
Added navigation to HighScoreActivity when player makes an incorrect guess

rchorngren committed:
Added localization throughout the GameActivity
Minor code cleanup

mirandamutka reviewed (changes requested):
The "another" button is missing localization. The rest looks good.

rchorngren committed:
Added localization to the placeholder function of "another button"
This button will be removed in the future for a more sophisticated way to continue playing the game

mirandamutka reviewed (approved):
Looks good!
